### PR TITLE
Add `open permalink to line` action

### PR DESCRIPTION
**Pull request by DanielMSchmidt**
Release Notes:

- Added `open permalink to line` action.

### CODE REVIEW

**Comment by DanielMSchmidt** [comment]
@cla-bot check

File: crates/editor/src/editor.rs [hunk 1]
```diff
@@ -8351,7 +8351,7 @@ impl Editor {
         }
     }
 
-    pub fn copy_permalink_to_line(&mut self, _: &CopyPermalinkToLine, cx: &mut ViewContext<Self>) {
+    fn get_permalink_to_line(&mut self, cx: &mut ViewContext<Self>) -> anyhow::Result<url::Url> {
```
Comment (maxdeviant) [inline crates/editor/src/editor.rs:8354]: We shouldn't need to qualify this, as the `Result` in scope is already `anyhow`'s:

```suggestion
    fn get_permalink_to_line(&mut self, cx: &mut ViewContext<Self>) -> Result<url::Url> {
```

Comment (SomeoneToIgnore) [reply crates/editor/src/editor.rs:8354]: I would not event slightly advocate for it in this repo (as it's far too gone to the other side of the spectrum), but nonetheless want to note: I generally really like when only `std::result::Result` gets unqualified as something coming from the prelude; the rest of the results seem to be much more readable when qualified.

**Comment by maxdeviant** [comment]
@DanielMSchmidt Thank you for the PR!

**Comment by DanielMSchmidt** [comment]
Thank you for fixing it, I was just about to make the changes :D

**Pull request merged**